### PR TITLE
corrected failing cuke and social buttons

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/helpers/publisher_helper.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def all_aspects_selected?(selected_aspects)
   def service_button(service)
     content_tag :div,
                 :class => "btn btn-link service_icon dim",
-                :title => "#{service.provider.titleize} (#{service.nickname})",
+                :title => z"#{service.provider.titleize} (#{service.nickname})",
                 :id => "#{service.provider}",
                 :maxchar => "#{service.class::MAX_CHARACTERS}",
                 :data  => {:toggle=>'tooltip', :placement=>'bottom'} do
```
Comment (jhass) [inline app/helpers/publisher_helper.rb]: as always, please update the hash syntax while touching ;)

Comment (AugierLe42e) [inline app/helpers/publisher_helper.rb]: Meh... This should have stayed untouched. That's a mistake :/

**Comment by jhass** [comment]
Character counter still needs a push downwards ;)

**Comment by AugierLe42e** [comment]
Oh ! What's this ? Is this for Twitter ?

**Comment by jhass** [comment]
All services that have a character limit.

**Pull request merged**